solutions/usaco-376.mdx

```diff
@@ -0,0 +1,83 @@
+---
+id: usaco-376
+source: USACO Bronze 2014 January
+title: Ski Course Design
+author: Juheon Rhee
+---
+
+[Official Analysis](http://www.usaco.org/current/data/sol_skidesign.html)
+
+## Implementation
+
+**Time Complexity:** $\mathcal{O}(N^2)$
+
+<LanguageSection>
+
+<PySection>
+
+```python
+with open('skidesign.in') as read:
+	n = int(read.readline())
+	elevation = [int(read.readline()) for _ in range(n)]
+
+ans = 10 ** 10
+
+for i in range(1, 83):
+	temp = 0
+	for j in elevation:
+		if j < i:
+			temp += (j - i) ** 2
+		elif j > i + 17:
+			temp += (i + 17 - j) ** 2
+	ans = min(ans, temp)
+    
+print(ans, file=open('skidesign.out', 'w'))
+```
+
+</PySection>
+
+<CPPSection>
+
+```cpp
+#include <bits/stdc++.h>
+using namespace std;
+
+int elevation[1001];
```

Comment:
(optional) just use a vector & dynamically size it

Reply:
bump

Reply:
bump

Reply:
no. i don't like vectors.

Reply:
elaborate?